fastapitableau/pages.py

```diff
@@ -148,24 +147,16 @@ def custom_get_swagger_ui_html(
     </header>
     <!-- END: Insert our header into the documentation -->
 
-    <!-- BEGIN: Embed swagger docs into our style containers -->
-    <div class="md-container">
-    <main class="md-main">
-      <div class="md-main__inner md-grid">
-        <div class="md-content">
-          <article class="md-content__inner md-typeset" style="margin-top: -2.7rem !important; margin-left: -5px;">
-    <!-- END: Embed swagger docs into our style containers -->
+    <!-- BEGIN: Small container to make positions consistent between this and other pages -->
```

Comment:
Thanks for doing this in a more consistent way.

I wasn't as clear when I was talking about it last night, but I _do_ agree with you that the slightly larger left margin looks better. I think that I was reacting to the narrow max width of the Swagger docs within our UI — I think padding the left and right sides evenly, rather than forcing it to be the same column width as everything else, would be acceptable too. :)